instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/BaseTracer.java

```diff
@@ -81,6 +82,9 @@ protected BaseTracer(OpenTelemetry openTelemetry) {
    */
   protected abstract String getInstrumentationName();
 
+  protected InstrumentationType getInstrumentationType() {
+    return InstrumentationType.NONE;
```

Comment:
we have to onboard Tracers to `InstrumentationType` to make `WithSpan` work well with instrumenter's suppression at least until `WithSpan` uses Tracers